src/AWS.Distro.OpenTelemetry.AutoInstrumentation/Plugin.cs

```diff
@@ -260,6 +256,11 @@ public void ConfigureTracesOptions(HttpClientTraceInstrumentationOptions options
                 return false;
             }
 
+            if (request.RequestUri?.AbsolutePath.Contains("/runtime/invocation/") == true)
```

Comment:
It's meant to be loose (or more generic). This is because we actually don't want to instrument any call that contains /runtime/invocation because this means it's coming from the dotnet runtime. Example is `http://169.254.100.1:9001/2018-06-01/runtime/invocation/next` or `http://169.254.100.1:9001/2018-06-01/runtime/invocation/9473f90d-bf8d-5d8b-9d46-a6c78e9d67ba/response`. I can use regex matching but what if I missed an API during testing, i'll also have to add it here so i decided it would be better to just match all instead.